tests/tracer/test_single_span_sampling_rules.py

```diff
@@ -119,6 +122,56 @@ def test_single_span_rules_do_not_tag_if_tracer_samples():
         assert_sampling_decision_tags(span, sample_rate=None, mechanism=None, limit=None, trace_sampling=True)
 
 
+def test_sampling_rule_init_config_multiple_sampling_rule_json_via_file(tmpdir):
+    file = tmpdir.join("rules.json")
+    file.write(
+        '[{"service":"xy?","name":"a*c"}, \
+            {"sample_rate":0.5,"service":"my-service","name":"my-name", "max_per_second":"20"}]'
+    )
+
+    with override_env(dict(DD_SPAN_SAMPLING_RULES_FILE=str(file))):
+        sampling_rules = get_file_json()
+        assert sampling_rules[0]["service"] == "xy?"
+        assert sampling_rules[0]["name"] == "a*c"
+
+        assert sampling_rules[1]["sample_rate"] == 0.5
+        assert sampling_rules[1]["service"] == "my-service"
+        assert sampling_rules[1]["name"] == "my-name"
+        assert sampling_rules[1]["max_per_second"] == "20"
+        assert len(sampling_rules) == 2
+
+
+def test_wrong_file_path(tmpdir):
+    """Test that single span sampling tags are not applied to spans that do not match rules via file"""
+    with override_env(dict(DD_SPAN_SAMPLING_RULES_FILE="data/this_doesnt_exist.json")):
+        if sys.version_info.major > 3:
```

Comment:
Avoid repeating the test code by setting a variable for the exception class we should expect.